subsys/net/openthread/rpc/client/ot_rpc_commissioning_client.c

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <ot_rpc_ids.h>
+#include <ot_rpc_types.h>
+#include <ot_rpc_common.h>
+
+#include <nrf_rpc/nrf_rpc_serialize.h>
+#include <nrf_rpc/nrf_rpc_cbkproxy.h>
+
+static void ot_rpc_thread_discover_cb_rpc_handler(const struct nrf_rpc_group *group,
+						  struct nrf_rpc_cbor_ctx *ctx, void *handler_data)
+{
+	otHandleActiveScanResult callback_slot;
```

Comment:
It's not a slot, it's already decoded callback, right?
```suggestion
	otHandleActiveScanResult callback;
```